src/modules/roc_core/target_posix/roc_core/format_time.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <stdio.h>
+#include <sys/time.h>
+#include <time.h>
+
+#include "roc_core/format_time.h"
+
+namespace roc {
+namespace core {
+
+bool format_time(char* buf, size_t bufsz) {
+    timeval tv;
+    if (gettimeofday(&tv, NULL) == -1) {
+        return false;
+    }
+
+    tm t;
+    if (!localtime_r(&tv.tv_sec, &t)) {
+        return false;
+    }
+
+    size_t off = strftime(buf, bufsz, "%H:%M:%S", &t);
```

Comment:
I think we need to specify the time format in doc because now it is impossible to understand the buffer size without looking into the source code.

Reply:
Why not? For now this is a small utility function used exclusively for logging. Implementing portable time formats would be non-trivial. And we don't need it so far.

Keeping the format implementation-defined would be much simpler. And that's why it's not documented.

Reply:
I think I misunderstood you. I was thinking about implementing portable configurable formats, but instead we can just declare a single fixed format that should be implemented for all platforms.

Done.